examples/auth-generate-token/page.md

```diff
@@ -0,0 +1,151 @@
+# Generate an Ably token to authenticate clients
+
+Use an Ably token to securely authenticate users within your application.
+
+Ably tokens enable you to grant users access to the Ably platform. They are used to issue time-limited credentials that securely authorize users, and manage which resources each user has access to.
+
+Authentication is the first step for an application to communicate with Ably. The application initializes a client and calls a predefined `authUrl` endpoint on your backend server. This endpoint uses an API key to request a token from Ably and returns it to the requesting client. When the token nears expiry, the `authUrl` is automatically invoked to request a new token.
+
+Authentication is implemented using [Ably core Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK provides authentication capabilities that are shared across all Ably products. The SDK includes purpose-built APIs that streamline pub/sub features while delivering Ably's performance guarantees and scaling capabilities.
```

Comment:
```suggestion
Authentication is implemented using [Ably core Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK provides the authentication mechanism that is utilized by all Ably products.
```

Then finish it off with whatever line we agree on regarding the Pub/Sub SDK (`The Pub/Sub SDK provides a set of flexible APIs capable of building any realtime application. It is powered by Ably's reliable and scalable platform.`) was my suggestion in the other PR.